vividus-plugin-azure-storage-account/src/main/java/org/vividus/azure/storage/blob/BlobStorageSteps.java

```diff
@@ -214,6 +214,22 @@ public void uploadBlob(String blobName, DataWrapper data, String containerName,
         createBlobClient(blobName, containerName, storageAccountKey).upload(BinaryData.fromBytes(data.getBytes()));
     }
 
+    /**
+     * Uploads the blob to the container. If blob already exists it will be replaced.
+     *
+     * @param blobName          The full path to the creating blob in the container.
+     * @param data              The data to store as blob.
+     * @param containerName     The name of the container to point to.
+     * @param storageAccountKey The key to Storage Account endpoint.
+     */
+    @When("I update blob with name `$blobName` and data `$data` to container `$containerName` of storage account "
```

Comment:
maybe we can use upsert instead of upload, it will describe the step purpose in the wording

Reply:
<img width="1295" alt="image" src="https://user-images.githubusercontent.com/30923336/161205187-b527a0ff-3692-4726-bbef-eb50b19cb35a.png">
:O